build.html

```diff
@@ -37,6 +37,7 @@
       };
     </script>
     <div data-script-insert="{{ id }}" class="hidden"></div>
+    <script type="text/javascript" src="{{{ asset "js/query-parser.js" }}}"></script>
```

Comment:
Nope. This asset should be listed in the widget.json configuration!

Reply:
I tried, but for some reason the LFD got loaded before it.. Will try again.

Reply:
@angelnikolov did you try adding it as first asset in the list?

Reply:
Of course, the issue isn't with that. Adding it as the first element in the list will put in @ the top of the `rendering.js` file for the LFD. However, the way the lfd code is inserted on the page is hardcoded in `build.html` file here as
```
    <div data-script-insert="{{ id }}" class="hidden"></div>
```
which later transpiles to `https://d2fx5g1ro9ok5z.cloudfront.net/widgets/com.fliplet.dynamic-lists/1.4.0/js/layout-javascript/small-card-code.js?_=1549623239` from Handlebars.
By the time this loads, the `rendering.js` file has not even been requested by `Fliplet.require` since that call is lower in the dom..

Reply:
I see. A few questions for you and @hcarneiro:
1) why is the script URL being printed out in the DOM? Is it because you only include a JS file to the selected templated without having to include all of them? If so, I assume you parse this somewhere in your code and inject the script as a script tag, correct? @angelnikolov given that, you just need to load the query-parser before/after such file (depending on what you need)

Reply:
That is correct, and that's exactly what the code is doing, by adding the query-parser before the file. Adding it to the assets in widget.json wouldn't work in this case, because it will always be after.
However I just thought of another way to load the specific layout's code and that's to just wrap the whole script loading in `Fliplet.then()`, please review it again.